src/odemis/acq/stream/_projection.py

```diff
@@ -52,6 +58,9 @@ def __init__(self, stream):
         # DataArray or None: RGB projection of the raw data
         self.image = model.VigilantAttribute(None)
 
+    def projectAsRaw(self):
```

Comment:
This needs a docstring explaining what's the purpose, and what the (typical) return.